matrix/Quaternion.hpp

```diff
@@ -83,8 +163,18 @@ class Quaternion : public Vector<Type, 4>
                sinPhi_2 * sinTheta_2 * cosPsi_2;
     }
 
-    Quaternion(Type a, Type b, Type c, Type d) :
-        Vector<Type, 4>()
+    /**
+     * Set from quaternion values
+     *
+     * Instance is set from quaternion values representing coordinate
```

Comment:
@LorenzMeier I've added it here. Additionally, the quaternion order is specified in the description at the top.